firmware/src/boards/cynthion_d11/apollo_board.h

```diff
@@ -52,15 +57,15 @@ enum {
  */
 enum {
 	FPGA_PROGRAM   = PIN_PA08,
-#if ((_BOARD_REVISION_MAJOR_ == 0) && (_BOARD_REVISION_MINOR_ < 6))
+#ifdef BOARD_HAS_SHARED_BUTTON
 	PROGRAM_BUTTON = PIN_PA16,
 	PHY_RESET      = PIN_PA09,
 #else
 	PROGRAM_BUTTON = PIN_PA02,
 	USB_SWITCH     = PIN_PA06,
 	FPGA_ADV       = PIN_PA09,
 #endif
-#if ((_BOARD_REVISION_MAJOR_ == 1) && (_BOARD_REVISION_MINOR_ > 2))
+#ifdef BOARD_HAS_FPGA_SYSCFG_PINS
```

Comment:
If `_BOARD_REVISION_DETECT_` is `1`, we're defining `FPGA_INITN` and `FPGA_DONE` for certain hardware revisions (r0.6 through r1.1) even though the pins don't really exist. If we're doing that, why not just define them for all hardware revisions?